cli/auth.go

```diff
@@ -143,7 +157,28 @@ func (c *viamClient) loginAction(cCtx *cli.Context) error {
 		return err
 	}
 
-	loggedInMessage(c.conf.Auth, false)
+	loggedInMessage(t, false)
+	return nil
+}
+
+// LoginWithAPIKeyAction is the corresponding Action for `login api-key`.
+func LoginWithAPIKeyAction(cCtx *cli.Context) error {
+	c, err := newViamClient(cCtx)
+	if err != nil {
+		return err
+	}
+	key := apiKey{
+		KeyID: c.c.String(loginFlagKeyID),
```

Comment:
```suggestion
		KeyID: cCtx.String(loginFlagKeyID),
```
[nit] can just pull from the `cCtx` here and below.